README.md

```diff
@@ -31,16 +31,16 @@ If the bot stops working, look at the logs with
 
     heroku logs -a sympy-bot
 
-(you need to install the `heroku` command line tools and have access to the
+(you need to install the `Heroku` command line tools and have access to the
```

Comment:
`heruku` is in backquotes (code) because it refers to the name of the command line tool, which is itself lowercase.